spec/hutch/worker_spec.rb

```diff
@@ -45,9 +45,17 @@
     end
 
     it 'sets up a subscription' do
-      expect(queue).to receive(:subscribe).with(manual_ack: true)
+      expect(queue).to receive(:subscribe).with(consumer_tag: %r(^hutch\-.{36}$), manual_ack: true)
```

Comment:
Ideally Hutch should loudly complain about prefixes that are longer, e.g. log an error and trim the prefix or fail to start.